embedded-minio/README.adoc

```diff
@@ -15,7 +15,7 @@
 ==== Consumes (via `bootstrap.properties`)
 * `embedded.minio.enabled` `(true|false, default is true)`
 * `embedded.minio.reuseContainer` `(true|false, default is false)`
-* `embedded.minio.dockerImage` `(default is 'minio/minio:latest')`
```

Comment:
Changed to specific version since `latest` version in the future may not be compatible with our setup.